resolver/resolver.cc

```diff
@@ -1017,28 +1016,31 @@ class ResolveConstantsWalk {
                 }
                 continue;
             }
+            auto idLoc = core::Loc(todo.file, id->loc);
```

Comment:
The diff looks somewhat annoying because i pulled this variable out so that clang-format didn't wrap lines in a weird way.